hil/ext/switches/nexus.py

```diff
@@ -38,13 +44,13 @@ class Nexus(Switch):
 
     @staticmethod
     def validate(kwargs):
-        schema.Schema({
+        Schema({
             'username': basestring,
             'hostname': basestring,
             'password': basestring,
-            'dummy_vlan': schema.And(schema.Use(int),
-                                     lambda v: 0 <= v and v <= 4096,
-                                     schema.Use(str)),
+            'dummy_vlan': And(Use(int),
+                              lambda v: 0 <= v and v <= 4096,
```

Comment:
while we are talking about vlan ranges, I am sure 0 shouldn't be included in this range. 
```
haas-loaner# show vlan id ?
  <1-3967,4048-4093>  VLAN ID 1-4094 or range(s): 1-5, 10 or 2-5,7-19
>
haas-loaner# show vlan id 0

```